tensorflow/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# install using pip from the whl file provided by Google
+
+if [ `uname` == Darwin ]; then
+    pip install https://storage.googleapis.com/tensorflow/mac/tensorflow-0.5.0-py2-none-any.whl
+fi
+
+if [[ (`uname` == Linux) ]]; then
+    pip install https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.5.0-cp27-none-linux_x86_64.whl
```

Comment:
This is OK, but why not build from source?  Looks like it would require one other package (Bazel): https://github.com/tensorflow/tensorflow/blob/master/tensorflow/g3doc/get_started/os_setup.md#detailed_install